python/jupyter-notebooks/ohsome_green_region.ipynb

```diff
@@ -0,0 +1,833 @@
+      "text/plain": [
+       "<Figure size 504x504 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "devdens = {'region':['Vancouver', 'Halifax', 'Edmonton', 'Ottawa'],\n",
+    "           'Δ density': [316130.09, 299203.41, 49647.97, 322071.36],\n",
+    "           'percent': [0.998060581278172, 1.0, 1.0, 0.9914301847182365]}\n",
+    "dev_density = pd.DataFrame(devdens)\n",
+    "\n",
+    "display (dev_density)\n",
+    "dev_density.plot.bar(0, figsize=(7,7), legend=False, subplots=True)\n",
+    "\n",
+    "#one way to calculate these values is:\n",
+    "#display(df_density)\n",
+    "#EdDiff = 49647.97 -  0.00\n",
+    "#EdPerc = EdDiff/49647.97\n",
+    "#display(EdDiff, EdPerc)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "banner-cream",
+   "metadata": {},
+   "source": [
+    "#### _Above you can see the Δ density and p%-values in numbers as well as figures. You can notice that the development for ALL regions was very good as the minimum percentuage of growth in density is in Ottawa with 99.143%_\n",
```

Comment:
I'd say the percentual visualization here is not really necessary as all values are almost or around 1, so it does not give any interesting impression.